network/p2p/p2p.go

```diff
@@ -216,7 +216,9 @@ func (n *p2pNetwork) UpdateSubnets(logger *zap.Logger) {
 	// TODO: this is a temporary fix to update subnets when validators are added/removed,
 	// there is a pending PR to replace this: https://github.com/bloxapp/ssv/pull/990
 	logger = logger.Named(logging.NameP2PNetwork)
-	for {
+	ticker := time.NewTicker(2 * time.Second)
+	defer ticker.Stop()
+	for range ticker.C {
```

Comment:
> The first tick will be delayed. Do we need it?

i'd prefer it it wasn't, but OTOH i'm not sure if there are any `activeValidators` already in the map on the first call? might be that we only call `Subscribe` (in validator.Start) a bit later, not sure

Reply:
maybe we should just reduce the tick duration to catch changes faster?

Reply:
We will do it in the refactor

Reply:
@nkryuchkov agree, but only disadvantage for now is that you have a 2 second delay since adding new validators until we subscribe to subnet, which is not too bad temporarily imo